src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -215,17 +216,18 @@ private void GroupFixes(Workspace workspace, IEnumerable<CodeFixCollection> fixC
                     var fixes = fixCollection.Fixes;
                     var fixCount = fixes.Length;
 
+                    Func<CodeAction, SuggestedActionSet> getFixAllSuggestedActionSet = codeAction =>
+                                CodeFixSuggestedAction.GetFixAllSuggestedActionSet(codeAction, fixCount, fixCollection.FixAllContext,
+                                    workspace, _subjectBuffer, _owner._editHandler);
+
```

Comment:
Yes, that is the reason this is a delegate passed down to SuppressionSuggestedAction ctor, which later invokes it for each nested action [here](http://source.roslyn.io/#Microsoft.CodeAnalysis.EditorFeatures/Implementation/Suggestions/SuppressionSuggestedAction.cs,69)